New and updated docs for HIS Designer for Logic Apps and 3270 Design Tool (#935)
MicrosoftDocs/biztalk-docs: his/core/application-integration-3270designer-1.md

---
title: 3270 Design Tool

> description: Design green-screen applications using the 3270 Design Tool.

Reviewer: ```suggestion
description: Learn about designomg green-screen applications using the 3270 Design Tool.
```

ms.prod: host-integration-server
ms.topic: conceptual
ms.date: 10/17/2023

#CustomerIntent: As a programmer, I need to design 3270 applications using the 3270 Design Tool.
---

# Get started with the 3270 Design Tool

> Development tool for the IBM 3270 connector for Azure Logic Apps.

Reviewer: ```suggestion
The 3270 Design Tool helps you generate the Host Integration Designer XML (HIDX) file that's required by the IBM 3270 connector in Azure Logic Apps.
```

---
title: 3270 Design Tool

> description: Design green screen applications using the 3270 Design Tool

Reviewer: ```suggestion
description: Design green-screen applications using the 3270 Design Tool.
```

> author: haroldcampos

Reviewer: No need to include author metadata as you're listed in the docfx.json file as the default author:

```suggestion
```

> ms.author: "hcampos"

Reviewer: No need to include author metadata as you're listed in the cocfx.json file as the default author:

```suggestion
```

> ms.prod: "host-integration-server"

Reviewer: ```suggestion
ms.prod: host-integration-server
```

ms.topic: conceptual

> ms.date: "10/17/2023"

Reviewer: ```suggestion
ms.date: 10/17/2023
```

> #CustomerIntent: As a Programmer I need to use the 3270 Design Tool to design 3270 applications.

Reviewer: ```suggestion
#CustomerIntent: As a programmer, I need to design 3270 applications using the 3270 Design Tool.
```

---

# Get started with the 3270 Design Tool

> Development tool for the IBM 3270 Connector for Azure Logic Apps.

Reviewer: ```suggestion
Development tool for the IBM 3270 connector for Azure Logic Apps.
```

> ## [The 3270 Design Tool](application-integration-3270designer-2.md)

Reviewer: ```suggestion
## [3270 Design Tool](application-integration-3270designer-2.md)
```

This tool helps you record the screens, navigation paths, methods, and parameters for the tasks in your application that you add and run as 3270 connector actions. The tool generates a Host Integration Designer XML (HIDX) file that provides the necessary metadata for the connector to use for driving your mainframe application.

> ## [Designing Metadata artifacts for 3270 Integration](application-integration-la3270apps.md)  

Reviewer: ```suggestion
## [Designing metadata artifacts for 3270 integration](application-integration-la3270apps.md)  
```

> 3270 Streams or green screen applications still run Mission Critical systems in Financial Services, Healthcare and many other industries. This section details the strategies to integrate with these applications using the 3270 Design tool.

Reviewer: ```suggestion
3270 streams or green screen applications still run mission-critical systems in financial services, healthcare, and many other industries. This section details the strategies to integrate with these applications using the 3270 Design tool.
```

## Related content

> - [The HIS Designer for Logic Apps](application-integration-ladesigner-1.md)

Reviewer: ```suggestion
- [HIS Designer for Logic Apps](application-integration-ladesigner-1.md)
```
